docs/engineering/code-development.md

```diff
@@ -1,78 +1,100 @@
-# Backend Development
+# Code Development / Text Editors
 
 ## Table of Contents
 
-* [General Principles](#general-principles)
+* [Popular Editors at CivicActions](#popular-editors-at-civicactions)
+  * [Editor Configuration for Coding](#editor-configuration-for-coding)
+* [General Coding Principles](#general-coding-principles)
 * [Coding Standards](#coding-standards)
-  * [Drupal specific-coding](#coding-standards)
+  * [Drupal specific coding](#coding-standards)
 * [Debugging](#debugging)
 * [Git Commits](#git-commits)
-* [Editor Configuration](#editor-configuration)
 * [Code Examples](#code-examples)
   * [Don't use continue in short loops](#continue)
-  * [Simplify Boolean Logic](#boolean)
+  * [Simplify Boolean Logic](#simplify-boolean-logic)
+* [Still have questions?](#still-have-questions)
 
-## <a name="general-principles"></a>General Principles
+## Popular Editors at CivicActions
+
+We use text editors to write documentation in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) and to develop code.
+
+* [Atom](https://atom.io/)
+* [Emacs](https://www.gnu.org/software/emacs/): Installed on OS X and Linux by default. Also available on OS X via [Homebrew](http://brew.sh/)
+* [Nano](https://www.nano-editor.org/): Installed on OS X and Linux by default. Also available on OS X via [Homebrew](http://brew.sh/)
+* [Sublime Text](http://www.sublimetext.com/)
+* [Vim](http://www.vim.org/download.php): Installed on OS X and Linux by default. Also available on OS X via [Homebrew](http://brew.sh/)
+
+### Editor Configuration for Coding
+
+* `TAB_WIDTH = 8` // the default for most editors and printers - this is the displayed width of tabs, but is irrelevant as there shouldn't be any actual tabs!
```

Comment:
Do these environment variables apply to all editors? I am not sure, but would guess not. It seems like this duplicates Drupal's documentation here also?